src/redux/data/exports/thunks.js

```diff
@@ -110,12 +117,42 @@ export function renderSavedExport(id) {
     const projectId = state.data.sessionData.session.project_id;
     const jwt = state.data.sessionData.session.token;
     const host = state.data.sessionData.host;
-    const encodedJwt = encodeURIComponent(jwt);
 
-    const downloadUrl = `${host}/project/${projectId}/export/${id}/rendered?jwt=${encodedJwt}`;
-    window.location = downloadUrl;
+    const downloadUrl = `${host}/project/${projectId}/export/${id}/rendered`;
+    const renderedResponse = await fetch(downloadUrl, {
+      method: "GET",
+      headers: {
+        Authorization: jwt,
+        Accept: "text/csv",
+      },
+    });
+
+    downloadFile(renderedResponse);
 
     //dispatch(setIsLoading(false));
     //dispatch(addNewSavedExport(result));
   };
 }
+
+function downloadFile(exportResponse) {
+  const defaultFilename = "export.csv";
+
+  exportResponse.blob().then((data) => {
+    const a = document.createElement("a");
```

Comment:
nit: Prefer meaningful names as the code will be minified and potentially obfuscated during build.

```suggestion
    const anchor = document.createElement("a");
```